edx_proctoring/models.py

```diff
@@ -321,6 +322,53 @@ def get_active_student_attempts(self, user_id, course_id=None):
 
         return self.filter(filtered_query).order_by('-created')  # pylint: disable=no-member
 
+    def get_user_onboarding_attempt(self, user_id, course_id=None):
```

Comment:
I'm a little wary of referring to this as an onboarding attempt. An attempt is an onboarding attempt by virtue of the backend used in the course supporting onboarding. It's not really a property on the attempt itself. The attempt is a proctored practice exam. There's nothing about the attempt itself to suggest it's onboarding. I think it creates some confusion to refer to practice proctored exam attempts as being onboarding attempts when we don't know whether the attempts are in an exam in a course that uses a vendor that supports onboarding.

Reply:
That makes sense. I can update the language to make it more generic.